DNN Platform/Skins/Aperture/src/scripts/menu.ts

```diff
@@ -1,4 +1,31 @@
 export const navFunctions = (() => {
+	// add event listeners to all anchor tags with a hash in href
+	const anchorsWithHash: NodeListOf<HTMLAnchorElement> = document.querySelectorAll('a[href^="/#"]');
+	anchorsWithHash.forEach(anchorWithHash => {
+		// listen for click event
+		anchorWithHash.addEventListener('click', event => {
+			// prevent default anchor click behavior
+			event.preventDefault();
```

Comment:
we shouldn't prevent default if the link is on another page, right?

Reply:
@bdukes good catch - I just pushed a commit to resolve this.